### ISSUE TITLE
Support datastore clusters

### ISSUE THREAD

**Issue by braunsonm** [vmware-samples/cloud-native-storage-self-service-manager#1]
Hello,

I might be understanding incorrectly but does this tool not support migrating between datastore **clusters**? It seems the tool only accepts specific datastores and not clusters for migration.

**Comment by gohilankit** [comment]
@braunsonm That's correct. The tool only accepts specific datastores for migration, and not datastore clusters.
